<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Convertor</string>
  </property>
  <property name="windowIcon">
   <iconset theme="test">
    <normaloff>ic.png</normaloff>ic.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>600</width>
     <height>500</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Georgia</family>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>421</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="toSQL_butt">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>170</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>To SQL</string>
    </property>
   </widget>
   <widget class="QPushButton" name="tocsv_butt">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>220</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>To CSV</string>
    </property>
   </widget>
   <widget class="QPushButton" name="brows_butt">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>161</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse file</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exit_butt">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>350</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
